core/src/main/scala/zio/logging/LoggerLike.scala

```diff
@@ -0,0 +1,63 @@
+package zio.logging
+import zio.{ UIO, ZIO }
+
+trait LoggerLike[-A] { self =>
```

Comment:
You missed a few improvements like `decorate` and so forth. I'd copy them over, they won't break anything else.

Reply:
added derive